include/ibamr/FESurfaceDistanceEvaluator.h

```diff
@@ -188,14 +185,15 @@ class FESurfaceDistanceEvaluator
     void collectNeighboringPatchElements(int level_number);
 
     /*!
-     * Compute signed distance from volume extracted boundary mesh.
+     * Store the side elements in a set for a volume mesh.
      */
-    void computeSignedDistanceVolExtractedBdryMesh(int n_idx, int d_idx);
+    void sideElemVolExtractedBdryMesh();
 
     /*!
-     * Compute signed distance from surface mesh.
+     * Get the closest point on a triangle for a given point P and the angle-weighted pseudo normal.
      */
```

Comment:
`\brief Compute the closest point on a triangle for a given Eulerian point P and obtain its corresponding angle-weighted pseudo-normal from the surface mesh elements.`.

Add `\brief` at the beginning of each function description.